Python/Product/PythonTools/PythonTools/InterpreterList/InterpreterListToolWindow.cs

```diff
@@ -545,14 +546,15 @@ class CondaEnvironmentManagerUI : ICondaEnvironmentManagerUI {
 
             var paths = GetPathEntries(view);
             var pathCmd = string.IsNullOrEmpty(paths) ? "" : string.Format("$env:PATH='{0};' + $env:PATH; ", paths);
-            var psi = new ProcessStartInfo("powershell.exe");
-            psi.Arguments = string.Join(" ", new[] {
-                "-NoLogo",
-                "-NoExit",
-                "-Command",
-                pathCmd + string.Format("(Get-Host).UI.RawUI.WindowTitle = '{0} environment'", view.Description)
-            }.Select(ProcessOutput.QuoteSingleArgument));
-            psi.WorkingDirectory = view.PrefixPath;
+            var psi = new ProcessStartInfo(Path.Combine(Environment.SystemDirectory, "WindowsPowerShell", "v1.0", "powershell.exe")) {
+                Arguments = string.Join(" ", new[] {
+            "-NoLogo",
+            "-NoExit",
+            "-Command",
+            pathCmd + string.Format("(Get-Host).UI.RawUI.WindowTitle = '{0} environment'", view.Description)
+            }.Select(ProcessOutput.QuoteSingleArgument)),
+                WorkingDirectory = view.PrefixPath
+            };
```

Comment:
Same comment as above. The older code was easier to read.

Reply:
Same as above.